Modele_prediction/Modele_regression/ConvNet/README.md

```diff
@@ -0,0 +1,141 @@
+# Convnet
+
+[comment]: <> (TODO: faire quelque chose de plus cute)
+
+We propose a convnet deeplearning network using electromencephatographic(EEG) data as input to predict the angle of different motors. These motors will be the main driving unit for an exosquelletton to help people attain with dysmorphie musculaire.  
```

Comment:
Could change EEG to EMG

exosquelletton -> exoskeleton

'to help people attain with dysmorphie musculaire' -> 'to help people suffering from muscular dystrophy'